docs/gen_ref_pages.py

```diff
@@ -30,7 +30,7 @@
             ident = ".".join(parts)
             to_write = f"::: {ident}"
             to_write += f"\n    rendering:"
-            to_write += f"\n        show_if_no_docstring: true"
+            to_write += f"\n        show_if_no_docstring: false"
```

Comment:
This disable code doc for functions without docstring

Reply:
GG